### PR TITLE
New methods to check mutiple mic support and local stats collecting

### PR DESCRIPTION
**Pull request by virtuacoplenny**

### CODE REVIEW

File: JitsiMediaDevices.js [hunk 1]
```diff
@@ -73,6 +74,13 @@ const JitsiMediaDevices = {
     },
 
     /**
+     * Returns true if it is possible to have multiple audio input streams.
```
Comment (saghul) [inline JitsiMediaDevices.js]: This documentation string could use a bit of an improvement. I guess what we mean here is *simultaneous* input devices open, right?

File: JitsiMediaDevices.js [hunk 2]
```diff
@@ -73,6 +74,13 @@ const JitsiMediaDevices = {
...
+     */
+    isMultipleAudioInputSupported() {
+        return !RTCBrowserType.isFirefox();
```
Comment (saghul) [inline JitsiMediaDevices.js]: 😞

File: JitsiMediaDevices.js (hunk 1 above)
Comment (virtuacoplenny) [inline JitsiMediaDevices.js]: Yes, you are correct. I've updated the comment.

File: modules/statistics/LocalStatsCollector.js [hunk 3]
```diff
@@ -139,4 +138,8 @@ LocalStatsCollector.prototype.stop = function() {
     }
 };
 
+LocalStatsCollector.isLocalStatsSupported = function() {
```
Comment (paweldomas) [inline modules/statistics/LocalStatsCollector.js]: missing JSDoc here ?

**Pull request merged**